### ISSUE TITLE
Create an accessibility statement

### ISSUE THREAD

**Issue by robby1066** [robby1066/keepposted-help#22]
# What's the problem you're hoping this new feature will solve? 
It's unclear how accessible Keep Posted is without digging into the application and finding out for yourself. Some attention has been paid to accessibility, but some parts of the app are inaccessible for use with certain assistive technologies.

People interested in using Keep Posted, but concerned about it's accessibility with different assistive technologies, should be able to get answers to their questions quickly and easily.

# Description of feature
A website accessibility statement is a message to your visitors about your commitment to making your website content accessible to everyone. Specifically, it’s often a single page on your website outlining the policies, goals, and current methods on making your website content and services accessible to people with disabilities, including visual, auditory, physical, speech, cognitive, language, learning, and neurological disabilities.

Accessibility statements are important for several reasons:

- Show your users that you care about accessibility and about them
- Provide them with information about the accessibility of your content
- Demonstrate commitment to accessibility, and to social responsibility

# Alternatives and workarounds
Currently, a person would need to explore the app on their own (extensively) to determine what the accessibility status is. Alternatively, they could reach out to Robby for an explanation.

# Additional context
https://www.w3.org/WAI/planning/statements/

**Comment by robby1066** [comment]
Because Keep Posted is both a tool for creating content and accessing it, a complete accessibility statement for Keep Posted should address how it conforms to both the:

[Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/)

and 

[Authoring Tool Accessibility Guidelines (ATAG)](https://www.w3.org/WAI/standards-guidelines/atag/)

**Comment by robby1066** [comment]
Published the Keep Posted accessibility statement at:

https://www.keepposted.io/accessibility-statement

Includes:
- a statement of commitment
- current ways accessibility is supported
- current limitations, and some ways I'd support accessibility if there were more resources
- conformance status with [WCAG 2.1—AA](https://www.w3.org/WAI/standards-guidelines/wcag/) and [ATAG 2.0](https://www.w3.org/WAI/standards-guidelines/atag/)
- Technical specifications (as they relate to accessible technologies)
- Feedback options
